src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -12,6 +12,18 @@
 					<p class="featured-loans__body">
 						Kiva empowers underserved people to achieve their
 						dreams by crowdfunding loans and unlocking capital.
+						<router-link
+							class="show-for-large featured-loans__body-link"
+							href="/lend-by-category"
```

Comment:
@BoulderBrains this is causing an error. Sorry I didn't catch this in my initial review. Should be:
`:to="'/lend-by-category'"`

Reply:
@BoulderBrains Also, while you're in this code....  The design calls for this text to be vertically centered in the section.

Reply:
Will have these changes up for review shortly.